### PR TITLE
dmenu: update to 5.2

### PR DESCRIPTION
**Pull request by OriginCode**
Topic Description
-----------------

Update dmenu, fix dbg package splitting problem.

Package(s) Affected
-------------------

`dmenu` 5.2

Security Update?
----------------

No

Test Build(s) Done
------------------

**Primary Architectures**

- [x] AMD64 `amd64`   
- [x] AArch64 `arm64`
    

**Secondary Architectures**

Architectural progress for "secondary," or experimental ports does not impede on merging of this topic.

- [x] Loongson 3 `loongson3`
- [ ] MIPS R6 64-bit (Little Endian) `mips64r6el`
- [x] PowerPC 64-bit (Little Endian) `ppc64el`
- [x] RISC-V 64-bit `riscv64`

Update(s) Uploaded to Stable
----------------------------

**Primary Architectures**

- [ ] AMD64 `amd64`   
- [ ] AArch64 `arm64`

**Secondary Architectures**

Architectural progress for "secondary," or experimental ports does not impede on merging of this topic.

- [ ] Loongson 3 `loongson3`
- [ ] MIPS R6 64-bit (Little Endian) `mips64r6el`
- [ ] PowerPC 64-bit (Little Endian) `ppc64el`
- [ ] RISC-V 64-bit `riscv64`

### CODE REVIEW

File: app-utils/dmenu/autobuild/defines [hunk 1]
```diff
@@ -5,3 +5,4 @@ PKGDES="A generic menu for X"
 
 ABTYPE=plainmake
 AB_FLAGS_SPECS=0
+ABSPLITDBG=0
```
Comment (MingcongBai) [inline app-utils/dmenu/autobuild/defines]: Why?

**Review by MingcongBai — CHANGES_REQUESTED** [review]

File: app-utils/dmenu/autobuild/defines (hunk 1 above)
Comment (OriginCode) [reply app-utils/dmenu/autobuild/defines]: 圖片

Comment (MingcongBai) [reply app-utils/dmenu/autobuild/defines]: Well, then it means that the flags weren't passed in properly, unless it's an ELF-less package.

**Review by MingcongBai — APPROVED** [review]

**Comment by OriginCode** [comment]
LGTM on amd64:
圖片

**Pull request merged**